scene/animation/easing_equations.h

```diff
@@ -402,4 +402,13 @@ static real_t out_in(real_t t, real_t b, real_t c, real_t d) {
 }
 }; // namespace back
 
+namespace spring {
+static real_t out(real_t t, real_t b, real_t c, real_t d) {
+	t /= d;
+	t = CLAMP(t, 0.0, 1.0);
+	t = (sin(t * Math_PI * (0.2f + 2.5f * t * t * t)) * pow(1.0f - t, 2.2f) + t) * (1.0f + (1.2f * (1.0f - t)));
+	return c * t + b;
+}
```

Comment:
Need to implement `in()` /  `in_out()` /  `out_in()` with the same way as Bounce.

Reply:
There are 11 copies of the exact same functions in that file, Perhaps there is a way to separate and reuse...